### ISSUE TITLE
Color banding

### ISSUE THREAD

**Issue by bugraduz** [xma/T480-Clover#57]
lilucpu = 8 and platform id method is not working in ventura. What can we do about this problem.  İs there any new spoofing method.

**Comment by xma** [comment]
up-to-date config.plist from this repo , from commit "macOS Ventura" : lilucpu=8 has been removed
colors are ok

**Comment by bugraduz** [comment]
lilucpu=8 didn't exist anyway in the config file. therefore color banding problem has been occur.

**Comment by xma** [comment]
I did not notice it,
could you post a screenshot with an artifact ?

**Comment by bugraduz** [comment]
![P_20190707_204501](https://user-images.githubusercontent.com/39997863/220880759-8e5f0566-d3a4-41ec-8beb-3a14c1b08c51.jpg)

**Comment by bugraduz** [comment]
same issue this photo
![P_20190707_204501](https://user-images.githubusercontent.com/39997863/220881668-15c13034-6430-4eff-bc9a-6de0e02072de.jpg)

And my t480's screen showing as Unkown Screen in system report
![photo](https://user-images.githubusercontent.com/39997863/220881918-0f1cf9b1-247d-465d-bc31-899c8190eb5a.png)

**Comment by xma** [comment]
your screenshot tell that "Device ID" is 0x591E

if you are using this repo config.plist 

         <key>device-id</key>
        <data>FlkAAA==</data>            

=> 0x5916

![image](https://user-images.githubusercontent.com/548909/220883505-2c08b80c-cebc-4199-823c-b34a3eb0b9e1.png)

**Comment by xma** [comment]
do you still have color issue with up-to-date config.plist ?

**Comment by bugraduz** [comment]
yes ı am using up to date config.plist. ı still have color banding issue. 
![foto1](https://user-images.githubusercontent.com/39997863/220942037-489f9aeb-e677-48d8-b628-b2e1534e2922.png)
![IMG_7500](https://user-images.githubusercontent.com/39997863/220942647-561182e9-817a-462a-a617-fd9c8d5be9d8.jpg)
![IMG_7498](https://user-images.githubusercontent.com/39997863/220942713-0bce3e55-b77b-4aa2-8df1-1918926aee25.jpg)

**Comment by xma** [comment]
I didn't noticed it before

You're right, others are telling the same issue
https://www.reddit.com/r/hackintosh/comments/wghnm2/horrible_colour_banding_on_the_internal_display/

so far no clear way to fix it, as skylake was removed in ventura

If you happen to have a clue, I would gladly test & integrate
